src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2844,7 +2848,9 @@ var projectUniqueNamesForBuildIntToUpdate
             IReadOnlyCollection<SourceRepository> primarySources,
             INuGetProjectContext nuGetProjectContext,
             VersionRange versionRange,
-            CancellationToken token)
+            CancellationToken token,
+            string newMappingID = null,
+            string newMappingSource = null)
```

Comment:
As mentioned a few times, CA1068 recommends cancellation tokens be the last parameter, which is a convention used throughout most (if not all?) libraries.